ArchUnitNETTests/Domain/PlantUml/TestDiagram.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace ArchUnitNETTests.Domain.PlantUml
+{
+    public class TestDiagram
+    {
+        private string _path;
+        private IList<string> _lines = new List<string>();
+
+        public TestDiagram(string path)
+        {
+            _path = path;
+        }
+
+        public static TestDiagram In(string path)
+        {
+            return new TestDiagram(path);
+        }
+
+        public ComponentCreator Component(string componentName)
```

Comment:
suggestion: rename to ComponentBuilder? That might improve consistency with other builders in the project